### ISSUE TITLE
Error with code blocks

### ISSUE THREAD

**Issue by bre7** [drkameleon/iswift#19]
Example:
```objectivec
void (^someBlockName)(NSString *someParam) = ^(NSString *someParam) {
    if (someParam) {
        // 1
    } else {
        // 2
    }
};
```

or

```objectivec
void (^someBlockName)(NSString *someParam);
someBlockName = ^(NSString *someParam) {
    // code
};
```

Expected result: Code block converted to closure

Result: Error - `Unexpected : ^`

**Comment by drkameleon** [comment]
Well, blocks *supported*. However: as an argument only - for now. Not declaring them. Though it's in my to-do list for the very next release (1.4).

To get what I mean, try this:

    id a = ^(NSString *someParam) {
        if (someParam) {
            // 1
        } else {
            // 2
        }
    };

**Comment by bre7** [comment]
Another example for inout parameters:
```objectivec
id aaa = ^(CGFloat *someParam) {
    *f = 9988 * pow(320.0, 4.0) * *someParam + 505.3;
};
```

**Comment by drkameleon** [comment]
> Another example for inout parameters:
> 
> id aaa = ^(CGFloat *someParam) {
>     *f = 9988 * pow(320.0, 4.0) * *someParam + 505.3;
> };

Hmm, Hmm, Hmm... I believe this is totally unrelated with blocks. But really good job spotting it. I'll have a look into it. Thanks!

**Comment by drkameleon** [comment]
The last (sub)issue you mentioned has been fixed: https://github.com/drkameleon/iswift/issues/20

Now, let's move on to the core one... :)

**Comment by drkameleon** [comment]
Issue fixed as of the upcoming 1.4.

-------

**Example:**

```objective-c
int main() {
    void (^someBlockName)(NSString *someParam) = ^(NSString *someParam) {
        if (someParam) {
            // 1
        } else {
            // 2
        }
    };
    
    void (^someBlockName)(NSString *someParam);
    someBlockName = ^(NSString *someParam) {
        // code
    };
}
```

**Output:**
```swift
func main() -> Int {
	var someBlockName: (String) -> () = { (someParam: String) in 
		if someParam {
		
		} else {
		
		}
	
	}
	var someBlockName: (String) -> ()
	someBlockName={ (someParam: String) in 
	
	}

}
```
